include/xsimd/arch/xsimd_avx512bw.hpp

```diff
@@ -223,7 +223,6 @@ namespace xsimd
                 }
             }
         }
-
```

Comment:
?

Reply:
My mistake, spurious change. I added an API there and then deleted.